### ISSUE TITLE
Way to import/export favorites

### ISSUE THREAD

**Issue by saschanaz** [syuilo/Misskey#9331]
## Summary

Currently there are ways to import lists of users but none for importing favorited notes.

**Comment by norz** [comment]
👍 This would be very nice to have imho: bookmarks is a nice wellness feature when browsing a timeline, but losing them when the server shuts down can be slightly painful 😬.  

(I’ve already went twice though this experience on Misskey instances 🤷‍♀️.)

**Comment by syuilo** [comment]
Export is implemented, but do we need to import as well?

**Comment by saschanaz** [comment]
Yes, to move the exported favorites to other account for example.
